Student.rb

```diff
@@ -1,6 +1,6 @@
 require './person'
 
-class Student
+class Student < person
```

Comment:
- [ ] (OPTIONAL) The `Student` class is inheriting from the `Person` class, so  it should be `class Student < Person`. Kindly correct this before merging :100: 
- [ ] (OPTIONAL) Kindly also add the `attr_accessor :name, :age` and `attr_reader :id` to your Person class so as to be able to access those variables  :+1: